### DISCUSSION TITLE
Main branch protection with pull requests and branching

### DISCUSSION THREAD

**Discussion by Ojani** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#51] [Reminders]
Rules should be added to the repo to disallow pushes to the main branch. Instead, pull requests should be made. Also, when working on a feature, it should be on its own branch.

**Comment by SaucyRexy** [comment]
That sounds good. I agree with implementing branch protection rules to prevent direct pushes to the main branch and using pull requests for merging. I believe we currently don’t have access to the GitHub branch protection feature, but I’ll speak to the professor to enable it.
